arm/Microsoft.Authorization/policyAssignments/parameters/parameters.json

```diff
@@ -14,15 +14,18 @@
     "parameters": {
       "value": {
         "tagName": {
-          "value": "Tag"
+          "value": "env"
         },
         "tagValue": {
-          "value": "Value"
+          "value": "prod"
         }
       }
     },
     "location": {
-      "value": "westeurope"
+      "value": "australiaeast"
+    },
+    "subscriptionId": {
+      "value": "20d6fbfe-b049-471c-95af-1369d14d0d45"
```

Comment:
remove subID

Reply:
I'll change to the sub ID used by the service principal

Reply:
fixed